### ISSUE TITLE
Create a separate repository with stand-alone examples

### ISSUE THREAD

**Issue by altavir** [mipt-npm/kmath#71]
Each module should contain full configuration.

**Comment by thomasnield** [comment]
I got a few already.

https://github.com/thomasnield/numky/tree/master/src/test/kotlin/org/nield/numky

**Comment by altavir** [comment]
@thomasnield Yeah, I know. I just figured that the build setup is not obvious and separate module for examples is not enough. Also we probably won't be able to keep documentation up to date with release version, so it is better to have a number of working examples. I will create a repository in a short time and then will welcome contribution of examples.
